docs/release_notes.rst

```diff
@@ -7,12 +7,18 @@
 .. role:: smaller
 
 
+On master, :small:`November 5, 2018`
```

Comment:
No need for a new entry, here, just update the date remove the comma and the statement about "deprecation"; add your name with "thanks to"